Update config-settings.rst (#3052)
mattermost/docs: source/administration/config-settings.rst

Users
~~~~~~~~~~~~~~~~~~~~~~~~~

> View and manage active and inactive users. Also provides the ability the admin to revoke all user sessions. 

Reviewer: ```suggestion
View and manage active and inactive users, and revoke all user sessions. 
```

| This feature's ``config.json`` setting is ``"TokenEndpoint": "https://login.microsoftonline.com/common/oauth2/v2.0/token"`` with string input.                       |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Guest Access (Beta)
~~~~~~~~~~~~~~~~~~~~~~~~~

Enable Guest Access
^^^^^^^^^^^^^^^^^^^

> **True**: Allow guest invitations to channels within teams.  Please see Guest documenation for more information. 

Reviewer 2: ```suggestion
**True**: Allow guest invitations to channels within teams. Please see Guest documentation for more information. 
```

| This feature's ``config.json`` setting is ``"TokenEndpoint": "https://login.microsoftonline.com/common/oauth2/v2.0/token"`` with string input.                       |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Guest Access (Beta)
~~~~~~~~~~~~~~~~~~~~~~~~~

Enable Guest Access
^^^^^^^^^^^^^^^^^^^

> **True**: Allow guest invitations to channels within teams. Please see `Guest Accounts documenation <>`_ for more information. 

Reviewer 2: ```suggestion
**True**: Allow guest invitations to channels within teams. Please see :doc:`Guest Accounts documentation <guest-accounts>`_ for more information.
```
